unix/vncserver/selinux/vncsession.fc

```diff
@@ -18,12 +18,19 @@
 #
 
 HOME_DIR/\.vnc(/.*)?      gen_context(system_u:object_r:vnc_home_t,s0)
-/root/\.vnc(/.*)?      gen_context(system_u:object_r:vnc_home_t,s0)
+HOME_DIR/\.config      gen_context(system_u:object_r:user_home_t,s0)
```

Comment:
You should not use types from other modules. Applies to multiple places in this PR.

Reply:
I see. I thought I have to specify what types should be used for parent directories and that using `vnc_home_t` will create them with a wrong context.